### ISSUE TITLE
Moving drop-down window when song changes

### ISSUE THREAD

**Issue by anirudh-k** [kmikiy/SpotMenu#14]
When a song is changed, and the length of "<Artist Name - Song Title>" text in the menu bar varies, the window moves left/right. This means that if I wish to press the next arrow again, I have to move my cursor. This would be fixed if the drop-down window were tied to the right edge of the text in the menu bar instead of the center of the text.

**Comment by kmikiy** [comment]
Wouldn't it look weird if the drop-down were tied to the right edge?

I've also noticed this by the way. 
Sadly, by default the drop-down is in the centre, which means that even if the text in the menubar changes just a little, then the drop-down repositions itself. I couldn't find any settings in the drop-downs API to make it stop from moving, so what I did was create a 1x1 pixel invisible view which it drops down from, and always recalculate that tiny views position to be where it was when the drop-down was opened. If the text becomes too small then, yes it will be repositioned sadly.
